CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+# 0.0.106 - 2022-03-30
+
+## API Updates
+ * Minimum supported rust version (MSRV) is now 1.41.1 (#1310).
```

Comment:
For now our user base is reasonable so we can follow quickly the still fast-moving rust ecosystem. Though in the future we might give a release or two of warning to our users, it can be a time-consuming task ?